migrations/6_relinquish_control.js

```diff
@@ -0,0 +1,77 @@
+require("babel-register");
+const controlAccounts = require("./accounts").default;
+
+const RoleBasedAccessControl = artifacts.require("RoleBasedAccessControl");
+const EuroToken = artifacts.require("EuroToken");
+
+// Needs to match contracts/AccessControl/RoleBasedAccessControl.sol:TriState
+const TriState = { Unset: 0, Allow: 1, Deny: 2 };
```

Comment:
this is repeated multiple times. I would suggest extracting it to separate file and exporting it from there.

Reply:
done!